lib/pensio_api/terminal.rb

```diff
@@ -12,6 +16,10 @@ def self.all
     def initialize(terminal_body)
       @raw = terminal_body
       @title = @raw['Title']
+      @country = @raw['Country'] rescue nil
+      @natures = @raw['Natures']['Nature'] rescue []
+      @currencies = @raw['Currencies']['Currency'] rescue []
+      @terminal_methods = @raw['Methods']['Method'] rescue []
```

Comment:
Why calling this "terminal_methods", if the rest do not have the "terminal_" prefix? I think it is important that we are consistent.

Reply:
@emicha Methods is a key word in Ruby so I had to add a prefix. Except title, all properties are newly added. I can add prefix to the newer ones but they'll still be inconsistent with the title property which can not be changed to backwards compatibility.